packages/readmodel/test/tenantUtils.ts

```diff
@@ -0,0 +1,373 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import {
+  getMockTenant,
+  getMockTenantMail,
+  getMockVerifiedTenantAttribute,
+} from "pagopa-interop-commons-test";
+import {
+  tenantVerifiedAttributeVerifierInReadmodelTenant,
+  tenantVerifiedAttributeRevokerInReadmodelTenant,
+  TenantSQL,
+  TenantMailSQL,
+  TenantFeatureSQL,
+  TenantCertifiedAttributeSQL,
+  TenantDeclaredAttributeSQL,
+  TenantVerifiedAttributeRevokerSQL,
+  TenantVerifiedAttributeSQL,
+  TenantVerifiedAttributeVerifierSQL,
+  tenantCertifiedAttributeInReadmodelTenant,
+  tenantDeclaredAttributeInReadmodelTenant,
+  tenantFeatureInReadmodelTenant,
+  tenantInReadmodelTenant,
+  tenantMailInReadmodelTenant,
+  tenantVerifiedAttributeInReadmodelTenant,
+  DrizzleReturnType,
+} from "pagopa-interop-readmodel-models";
+import {
+  AttributeId,
+  CertifiedTenantAttribute,
+  DeclaredTenantAttribute,
+  DelegationId,
+  ExternalId,
+  generateId,
+  Tenant,
+  tenantAttributeType,
+  TenantFeature,
+  TenantFeatureCertifier,
+  TenantFeatureDelegatedConsumer,
+  TenantFeatureDelegatedProducer,
+  tenantFeatureType,
+  TenantId,
+  tenantKind,
+  TenantMail,
+  TenantRevoker,
+  tenantUnitType,
+  TenantVerifier,
+  VerifiedTenantAttribute,
+  WithMetadata,
+} from "pagopa-interop-models";
+import { eq } from "drizzle-orm";
+import { tenantReadModelServiceBuilder } from "../src/tenantReadModelService.js";
+import { readModelDB } from "./utils.js";
+
+export const tenantReadModelService =
+  tenantReadModelServiceBuilder(readModelDB);
+
+export const initMockTenant = (
+  isTenantComplete: boolean = true
```

Comment:
```suggestion
{
  isTenantComplete = true,
}: {
  isTenantComplete: boolean;
}
```
I would use a named boolean in order to better clarify what's the boolean inside